### ISSUE TITLE
[Feature request] Disconnected wired headphones gate

### ISSUE THREAD

**Issue by rahim21** [KieronQuinn/TapTap#159]
I use tap tap to play/pause my music and it's working great.. but I don't want it to start out loud in public by an unintentional tap

**Comment by Martin5001** [comment]
(Not just wired headphones, bluetooth too please.)

**Comment by 4shutosh** [comment]
Two gates can be added : 
* If headphones are connected
* If device is connected to a bluetooth device

**Comment by rahim21** [comment]
Isn't -if headphones disconnected possible ?

**Comment by 4shutosh** [comment]
@rahim21 Yeah I got your issue please wait some days or untll next release it should be solved.

**Comment by rahim21** [comment]
Thanks you're a life saver !
